components/ui/AppLink.vue

```diff
@@ -1,13 +1,17 @@
 <template>
   <component
-    :is="isInternal(url) ? 'nuxt-link' : 'a'"
+    :is="!isStatic && isInternal(url) ? 'nuxt-link' : 'a'"
     :href="url"
-    :to="isInternal(url) ? url : null"
+    :to="!isStatic && isInternal(url) ? url : null"
     :rel="isExternal(url) ? 'noopener' : null"
     :target="isExternal(url) ? '_blank' : null"
+    :title="title"
     @click="segment && $trackClickEvent(segment)"
   >
-    {{ label }}
+    <template v-if="label">
```

Comment:
The link already accepts general content in the `slot`, this shortcut is not necessary or it can be achieved by extension with a wrapper component. Leave the slot only, please.